Sources/Shared/AuthService.swift

```diff
@@ -23,11 +23,16 @@ import Foundation
 
   // MARK: - Initialization
 
-  public init(name: String, config: AuthConfig) {
+  public init(name: String, config: AuthConfig, locker: Lockable? = nil) {
```

Comment:
Yah that could also work I think. But does it have access to `name` in the label?